thirdeye-ui/src/app/components/alert-wizard-v2/alert-template/alert-template.utils.ts

```diff
@@ -115,3 +118,25 @@ export function createNewStartingAlert(): EditableAlert {
         },
     };
 }
+
+type DefaultValue = NonNullable<MetadataProperty["defaultValue"]>;
+
+export const getDefaultProperties = (
+    alertTemplate: AlertTemplate | null
+): Record<string, DefaultValue> => {
+    if (!alertTemplate) {
+        return {};
+    }
+
+    const oldDefaultProperties = alertTemplate.defaultProperties || {};
+    const newDefaultProperties: Record<string, DefaultValue> = Object.assign(
+        {},
+        ...(alertTemplate.properties || [])
+            .filter(({ defaultValue }) => !!defaultValue)
+            .map(({ name, defaultValue }) => ({
+                [name]: defaultValue,
+            }))
+    );
+
+    return { ...oldDefaultProperties, ...newDefaultProperties };
+};
```

Comment:
Please provide test cases for this function

Reply:
Added a testcase